### DISCUSSION TITLE
Do I need to set manual control before turning light on?

### DISCUSSION THREAD

**Discussion by dekiesel** [basnijholt/adaptive-lighting#301] [Q&A]
Hi,

What I am trying to do: I wrote an automation (in appdaemon) that reacts to a button press and then checks the weatherforecast for the next x hours. If it's going to rain then it turns on a light and (in the same call) sets the color to red. If it doesn't rain it sets the color to green. After 5 sec the light is returned to its former state.

Problem: The color of the light changes for a fraction of a second to the correct color but is then taken over by adaptive-lighting (when it was off previously).

I am not sure I understand the manual control part correctly, so: Do I need to manually turn off AL for the light I am trying to change or is there a switch I can flip somewhere to always detect changes, even when the light is off?

Thanks!

**Comment by broyuken** [comment]
Yes set manual control before you turn it on, then turn manual control off after 5 seconds and AL will make it the right color temp
